supernode/server/server.go

```diff
@@ -69,17 +71,34 @@ func New(cfg *config.Config) (*Server, error) {
 		return nil, err
 	}
 
+	haMgr, err := ha.NewManager(cfg)
+	if err != nil {
+		return nil, err
+	}
+
 	return &Server{
 		Config:       cfg,
 		PeerMgr:      peerMgr,
 		TaskMgr:      taskMgr,
 		DfgetTaskMgr: dfgetTaskMgr,
 		ProgressMgr:  progressMgr,
+		HaMgr:        haMgr,
 	}, nil
 }
 
 // Start runs
 func (s *Server) Start() error {
+	if s.Config.UseHa == false {
+		if changOk := s.HaMgr.HaCompareAndSetSupernodeState(0, 2); changOk == false {
+			fmt.Println("supernode change fail")
+		}
+		fmt.Println("the syetem don not use ha,now the supernode status change from initial to active")
```

Comment:
Even though failed to `HaCompareAndSetSupernodeState`, and you still want to output this sentence?

Reply:
sorry,it is a temporary state,I will handle these errors and exceptions in the future

Reply:
If that, you can record a TODO comment here. That will be more clear for you or someone else.